src/psc/app.py

```diff
@@ -1,8 +1,8 @@
 """Provide a web server to browse the examples."""
 import contextlib
+from collections.abc import Iterator
```

Comment:
This is a really good catch. I always get this wrong. I probably should write some flake8 extension that complains. 😀

Reply:
I think this is specifically to latest python versions right?
I was used to do `from typings import blah` haha